source/core/slang-test-tool-util.cpp

```diff
@@ -37,6 +37,19 @@ namespace Slang
     }
 }
 
+/* static */bool TestToolUtil::hasDeferredStdLib(Index argc, const char*const* argv)
+{
```

Comment:
I will continue to register my complaint that I don't like the test-runner looking at the command line of tests to figure out things about them. There's not a great alternative right now, but if/when we get around to a grand refactor of `slang-test`, I think that one of the big goals will be making it not have to know/understand the policy of each and every test type it supports.